_posts/2023-05-01-whats-new-2023-05.md

```diff
@@ -28,7 +28,7 @@ We've got bigger plans in the long run for this simulator, and we're actively wo
 
 The main scenario in the experiment is pretty much done. It was our first time creating roundabouts, so we were anxious about making it both look and feel right. We're very happy with the result, [RoadRunner](https://www.mathworks.com/products/roadrunner.html) is truly an amazing tool to use.
 
-Since SIMAX is undergoing maintenance, it was also an opportunity to start testing this experiment with its real "production" hardware. It was running fine on a single desktop computer with a single viewport; but how about on an [nDisplay](/ndisplay/) cluster with 8 computers, 12 viewports, covering almost 360° of horizontal field of view plus [pesky mirrors](/mirrors/)? Well, everything runs fine! No major hitches or performance drop, we pretty much have the same performance on the cluster as we do on desktop.
+Since SIMAX is undergoing maintenance, it was also an opportunity to start testing this experiment with its real "production" hardware. It was running fine on a single desktop computer with a single viewport; but how about an [nDisplay](/ndisplay/) cluster with 8 computers, 12 viewports, covering almost 360° of horizontal field of view plus [pesky mirrors](/mirrors/)? Well, everything runs fine! No major hitches or performance drop, we pretty much have the same performance on the cluster as we do on desktop.
```

Comment:
Hmm not sure, mine seems valid here, no?

Reply:
Yeah also, I must have suggested that while reviewing the typos PyCharm found, not having the full sentence in mind anymore...